frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts

```diff
@@ -341,6 +341,11 @@ export function determineAllowedModes(settings?: Settings): {
     allowedModes,
   };
 }
+
+export function getMaximumSegmentIdForLayer(dataset: APIDataset, layerName: string) {
+  return getDefaultIntensityRangeOfLayer(dataset, layerName)[1];
```

Comment:
I think intensity range should be renamed to value range in this function, because intensity doesn't make sense in the context of segmentations.

Reply:
Is this what you intended? or should I rename the whole function `getDefaultIntensityRangeOfLayer` to something like `getDefaultValueRangeOfLayer`?

Reply:
> or should I rename the whole function getDefaultIntensityRangeOfLayer to something like getDefaultValueRangeOfLayer?

This is what I meant. Sorry that my initial comment wasn't clear enough :see_no_evil:

Reply:
This still needs a renaming, right?

Reply:
see my comment below please :arrow_down: